### DISCUSSION TITLE
Latest macOS works on IvyBridge?

### DISCUSSION THREAD

**Discussion by HubKing** [notAperson535/OneClick-macOS-Simple-KVM#24] [Q&A] [answered]
When I searched Google, the first result, a GitHub project, required AVX2 to run Mojave or later. AVX2 is supported since Haswell. My Linux server has an IvyBridge CPU, so it has no AVX2. The reason why I am trying to run a macOS is to compile iOS projects remotely (development is done on a non-macOS computer). So, graphics acceleration or sound, etc are not needed. In this case, does this VM work without AVX2?

**Comment by tutacat** [comment, marked as answer]
https://www.reddit.com/r/hackintosh/comments/gc1n3o/comment/fp94pq8/?utm_source=reddit&utm_medium=web2x&context=3

If macos runs on bare hardware 2nd gen (ivybridge), it will definitely run on a VM.

**Reply by HubKing** [reply]
I do not think I fully understood the second sentence..... but I guess it would work. I wondered if I have to upgrade the CPU/mainboard to run Majave or later on my Linux server, but maybe I don't have to.

**Reply by tutacat** [reply]
The comment linked above, says that someone ran macos (hackintosh) on a 2nd gen intel processor

If macos (hackintosh) runs on it directly, it should support all the features to run it, and mean the processor would also work in a VM.

**Reply by HubKing** [reply]
No, by "second" I meant "Be aware that github issues is for issues with a project, and this would be a problem with macos/qemu, if it was a problem.", excluding the link.

**Reply by notAperson535** [reply]
> No, by "second" I meant "Be aware that github issues is for issues with a project, and this would be a problem with macos/qemu, if it was a problem.", excluding the link.

@HubKing they are just saying to make a github issue next time
